stories/core/ComboBox.stories.tsx

```diff
@@ -392,13 +398,19 @@ export const WithStatus: Story<Partial<ComboBoxProps<string>>> = (args) => {
   const options = React.useMemo(() => countriesList, []);
 
   return (
-    <ComboBox
-      options={options}
-      inputProps={{ placeholder: 'Select a country' }}
-      onChange={(value: string) => action(value ?? '')()}
-      status='negative'
-      {...args}
-    />
+    <InputGroup
+      status={args.status ?? 'negative'}
+      message='Country is required.'
+    >
```

Comment:
Really don't like this hack. Can't we make a new `StatusMessage` component or something?

Reply:
hmm...I dont know. I guess we can, but then what to use when...? InputGroup it not needed anymore..?

Reply:
InputGroup would still be valid for these use cases
![image](https://user-images.githubusercontent.com/9084735/146043678-4c0868c2-188a-459d-993e-c7f3bf02bbd7.png)

Reply:
But I think you could have same without it too. Just put `Label`, few `Checkbox` and this new `StatusMessage` component at the end.

Reply:
InputGroup adds a bit of gap between the checkboxes by setting min-height

Reply:
and its only helpful if you have checkbox without label...
idk, tricky situation and not sure what would be the best here..

Reply:
At first I picked only `message` prop from `InputContainer`, but.. It does not handle icon and we are already using icon, so that means we cannot have 2 icons (inline and below) at the same time with current implementation. So I guess this is also not an option, cause just message prop mostly used on status messages, IMO.
So I guess we need to rethink whole thing and decide what we wanna do with this situation.